packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -182,6 +182,10 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
     ) {
         super();
 
+        if (dataStoreContext.id.includes("/")) {
+            throw new UsageError(`Data store context ID cannot contain slashes: ${dataStoreContext.id}`);
```

Comment:
I don't think we need this. Data store context would have already verified this. If we want to keep this, it should be an assert rather than a usage error.